humanguide/layout/05-dialogues.rst

```diff
@@ -0,0 +1,24 @@
+Dialogues
+=========
```

Comment:
Oh dear, it's UK English. Most of the documentation is US English, shall we change that here?

Reply:
:)